Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -329,6 +329,10 @@ void Renderer::DrawDebugText()
       ar_text = "Auto";
       break;
     }
+    std::string audio_text;
+    audio_text = SConfig::GetInstance().m_IsMuted ?
```

Comment:
You don't need to split declaration and assignment here. This can just be:

    const std::string audio_text = SConfig::GetInstance().m_IsMuted ? ...

(unrelated to your PR, but it's pretty silly how m_IsMuted is part of SConfig.)